src/Compatibility/Core/src/iOS/Renderers/NavigationRenderer.cs

```diff
@@ -1712,5 +1712,5 @@ protected override void Dispose(bool disposing)
 		}
 	}
 
-	internal class UpdateToolBarButtonsMessage { }
+	internal sealed record UpdateToolBarButtonsMessage { }
```

Comment:
Nit: empty messages don't need to be records, they can just be normal classes. There is not need for the additional code generated by the record syntax here, since there are no members that will ever need to be compared 🙂
Same for the other empty record below (`CloseContextActionsMessage`).

Reply:
Fixed, thanks!